help/sources/tutorials/ui/create/consent-and-preferences/didomi.md

```diff
@@ -0,0 +1,167 @@
+| **Endpoint URL**          | The final Adobe connector URL (provided at the end of the setup). | ✅           | `https://dcs.adobedc.net/collection/your-adobe-endpoint-id`                         |
+
+4. Configure the webhook options.
+
+| **Field**           | **Description**                                                                                                                                                                                                                                                                                                                                                                                      | **Required** | **Value**                                                                                                     |
+| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ | ------------------------------------------------------------------------------------------------------------- |
+| **Request Headers** | Custom headers for the webhook. Make sure to include the `x-adobe-flow-id` (provided later).                                                                                                                                                                                                                                                                                                         | ✅           | `{"Content-Type": "application/json", "Cache-Control": "no-cache", "x-adobe-flow-id": "your-flow-id"}`        |
+| **Flatten**         | Must be checked. Ensures that the webhook data is sent as a flat object.                                                                                                                                                                                                                                                                                                                             | ✅           | Enabled                                                                                                       |
+| **Event Types**     | Select the specific group of Didomi events (`event.*` or `user.*`) that should trigger the webhook. Use `event.*` to track consent or preference changes, and use `user.*` to track user profile updates. This selection is required to ensure that only compatible events are sent to Adobe. Adobe supports only one schema per dataflow, so selecting both event types can cause ingestion errors. | ✅           | `Event.created`, `Event.updated`, and `Event.deleted` — or `User.created`, `User.updated`, and `User.deleted` |
```

Comment:
Little bit of a nitpick here but can we clean up the table formatting here to look like the other ones?